elastic/logs/operations/esql.json

```diff
@@ -1,3 +1,23 @@
+    {
+      "name": "enable_query_cache",
+      "include-in-reporting": false,
+      "operation-type": "raw-request",
```

Comment:
We have a dedicated `put-settings` runner for this purpose: https://esrally.readthedocs.io/en/latest/track.html?highlight=cluster%20settings#put-settings 

To complicate things further, the tracks need to be runnable against both flavors of ES (serverless + stateful), and our current conditional approach can be found here:
https://github.com/elastic/rally-tracks/blob/a1cf09881d0a97c19c9aa284f86ab2453d49fd15/geonames/challenges/default.json#L62-L70

Reply:
I looked at put-settings and it was only for cluster settings, not index settings, so it seemed like it could not handle this case.

Reply:
OK, I've pushed a commit that should do this. Could you verify?

Reply:
Local testing shows that this turns off _search queries for my local elasticsearch. I need a condition that only turns it off for serverless.

Reply:
The condition related to serverless would seem to be:

    if build_flavor != "serverless" or serverless_operator == true

However, the first predicate sounds like 'not serverless', while the second sounds like 'is serverless'. I'll push a commit with this condition, but would like confirmation on it's validity.

Reply:
This latest predicate passed local stateful tests. And the CI passed it and you gave it a positive review, so I'll merge it now. If we want to refine the predicate, we can make another PR.

Reply:
> However, the first predicate sounds like 'not serverless', while the second sounds like 'is serverless'. I'll push a commit with this condition, but would like confirmation on it's validity.

It should be read as "execute only if running against non-serverless or with serverless operator privileges".